black/svg/1F680.svg

```diff
@@ -1,12 +1,10 @@
-    <path fill="none" stroke="#000000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" stroke-width="1.9493" d="M40.03,65.7798c-0.01,0.06-0.01,0.1-0.01,0.1h-8.11c0,0,0-0.04-0.0099-0.11c-0.05-0.68-0.27-4.18,0.53-5.93v-2.4 c-0.02-5.92-0.01-16.68,0.17-24.21v-0.0099c0.07-3.32,0.1801-6.01,0.34-7.39c0.66-5.91,3.02-6.79,3.02-6.79s2.37,0.88,3.04,6.79 c0.15,1.36,0.26,3.99,0.34,7.23c0.18,7.54,0.19,18.41,0.17,24.38v2.43C40.29,61.6298,40.07,65.1198,40.03,65.7798z"/>
+    <path id="path5860" d="m48.405 29.49c-3.2761 0-5.941-2.6641-5.941-5.9392 0.000001-3.2751 2.6649-5.9392 5.941-5.9392 3.2761-0.000002 5.941 2.6641 5.941 5.9392s-2.6649 5.9392-5.941 5.9392zm0-9.8987c-2.1839 0.000001-3.9607 1.7757-3.9607 3.9595 0 2.1837 1.7768 3.9595 3.9607 3.9595 2.1838 0 3.9607-1.7758 3.9607-3.9595 0-2.1838-1.7769-3.9595-3.9607-3.9595z"/>
+    <path id="path5029" stroke-linejoin="round" d="m20.653 45.063c-1.678 0.70826-3.2222 1.7475-4.5331 3.0508-3.1581 3.1631-4.6517 7.5594-3.9703 11.687 4.128 0.6762 8.5221-0.81961 11.683-3.9769 1.3043-1.3104 2.3446-2.8541 3.0537-4.5318" stroke="#000" stroke-linecap="round" stroke-miterlimit="10" overflow="visible" stroke-width="1.949" fill="none"/>
+    <path id="path6527" stroke-linejoin="round" d="m14.923 35.749a25.505 28.104 62.551 0 0 -1.9691 2.0383 28.101 25.507 44.912 0 1 9.7604 4.6784 4.9877 4.9862 0 0 1 0.84351 -1.1281 4.9877 4.9862 0 0 1 2.5417 -1.3598 25.507 28.101 45.088 0 1 3.293 -11.814 25.507 28.101 45.088 0 1 0.0056 -0.01117 25.505 28.104 62.551 0 0 -7.6405 2.7434 25.505 28.104 62.551 0 0 -6.8347 4.8529z" stroke="#000" stroke-linecap="round" stroke-miterlimit="10" overflow="visible" stroke-width="1.949" fill="none"/>
+    <path id="path6529" stroke-linejoin="round" d="m31.973 45.839a4.9877 4.9862 0 0 1 -1.3616 2.5507 4.9877 4.9862 0 0 1 -1.1452 0.85164 28.101 25.507 44.912 0 1 4.6952 9.7506 25.509 28.099 27.593 0 0 2.0417 -1.9685 25.509 28.099 27.593 0 0 4.8516 -6.8326 25.509 28.099 27.593 0 0 2.7442 -7.6354 25.507 28.101 45.088 0 1 -11.826 3.2837z" stroke="#000" stroke-linecap="round" stroke-miterlimit="10" overflow="visible" stroke-width="1.949" fill="none"/>
+    <path id="path6579" stroke-linejoin="round" d="m31.83 43.345c0.26961 0.8863 0.2506 1.6919 0.13707 2.5245 7.0759-0.36106 13.993-3.5031 19.243-8.7412 6.7106-6.7215 9.8836-16.063 8.4351-24.834-8.7712-1.4365-18.108 1.742-24.823 8.4508-5.2322 5.2509-8.3679 12.164-8.7242 19.234 0.94127-0.19073 1.8984-0.09419 2.7693 0.23874" stroke="#000" stroke-linecap="round" stroke-miterlimit="10" overflow="visible" stroke-width="1.949" fill="none"/>
+    <path id="path6463" stroke-linejoin="round" d="m37.072 34.196-0.000161 0.000009c-2.4156 1.2183-4.6724 2.7626-6.6996 4.5844-2.0849 1.8911-3.9 4.0556-5.3844 6.4211-0.50392 0.80311-0.96842 1.6273-1.3917 2.4694" stroke="#000" stroke-linecap="round" stroke-miterlimit="10" overflow="visible" stroke-width="1.949" fill="none"/>
```

Comment:
can you export the SVG's as `optimised` with `Remove unused IDs`? I think these are an artifact of that.

Reply:
I meant to add this on the file in /src/... I know you didn't directly create this file